<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AgoraStudentWidget</class>
 <widget class="QWidget" name="AgoraStudentWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1089</width>
    <height>793</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AgoraStudentWidget</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,30">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QWidget" name="top_widget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget{
background: #E5E5F0;
}
#top_widget{
border-left:1px solid #DBDBEA;
border-right:1px solid #DBDBEA;
border-bottom:1px solid #DBDBEA;
border-top:1px solid #DBDBEA;
border-top-left-radius:10px;
border-top-right-radius:10px;
}

</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="20,1,20,0,1">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="ClassroomNameLabel">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="ChangeSizePushButton">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
border-image: url(&quot;:/image/resource/icon-max@2x.png&quot;);
}

QPushButton:hover{
border-image: url(&quot;:/image/resource/icon-max-hover.png&quot;);
}

QPushButton:checked{
border-image: url(&quot;:/image/resource/icon-min@2x.png&quot;);
}

QPushButton:checked:hover{
border-image: url(&quot;:/image/resource/icon-min-hover@2x.png&quot;);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ExitPushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#ExitPushButton {border-image:url(&quot;:/image/resource/icon-exit@2x.png&quot;);}

#ExitPushButton:hover{
border-image:url(&quot;:/image/resource/icon-exit-hover@2x.png&quot;);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">
#widget{

border-bottom-left-radius:10px;
border-bottom-right-radius:10px;
border-top:1px solid #DBDBEA;
}
</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../agora_edu_demo.qrc"/>
 </resources>
 <connections/>
</ui>
